fastlane_core/lib/fastlane_core/helper.rb

```diff
@@ -47,6 +56,10 @@ def self.is_ci?
       ci?
     end
 
+    def self.is_bundler?
```

Comment:
We don't actually need this method, we only have some of those duplicates to support legacy code bases. Let's remove this and stick with `bundler?`